Automattic/wp-calypso
# Theme showcase: Reuse showcase header for the install theme page

#### Proposed Changes

Currently, the header install theme page looks off: it is somehow missing the top margin/padding. This PR fixes that.

While I'm touching the page, I'm taking the opportunity to make the header uniform with the theme showcase page as well, by refactoring and reusing the header component.

Before

<img width="1325" alt="image" src="https://user-images.githubusercontent.com/1525580/211819201-b1faea5e-9abc-4fc3-899c-2920503d60ea.png">

After

<img width="1329" alt="image" src="https://user-images.githubusercontent.com/1525580/211819083-69534c5c-17c2-4f43-81ba-349b3385cdfc.png">


#### Testing Instructions

* Go to `/themes/upload/{siteSlug}`
* Verify that the header now looks good.
* Verify that you can actually install a theme.
* Verify that the header in the theme showcase page is still correct (compare with production's).

#### Pre-merge Checklist





Related to:

- https://github.com/Automattic/wp-calypso/issues/71837
